src/components/groups/group.middlewares.ts

```diff
@@ -18,17 +19,35 @@ export const joinGroup = asyncWrapper(async (req: Request, res: Response, next:
   // We only join the group if it is not full already
   if (group.doNotDisturb && (user.id !== group.ownerId)){
     sendMessage(res, 'Ez egy privát csoport!')
+    res.redirect(`/groups/${req.params.id}`)
   } else if (group.users?.find(it => it.id === user.id)) {
     sendMessage(res, 'Már tagja vagy ennek a csoportnak!')
+    res.redirect(`/groups/${req.params.id}`)
   } else if ((group.users?.length || 0) >= group.maxAttendees) {
     sendMessage(res, 'Ez a csoport már tele van!')
+    res.redirect(`/groups/${req.params.id}`)
```

Comment:
We either should have a return statement after the redirects or move the `next()` function call in line 30 into the else block.